Aquality.Appium.Mobile.Template/Screens/WebLogin/IOSLoginScreen.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Aquality.Appium.Mobile.Applications;
+using Aquality.Appium.Mobile.Screens.ScreenFactory;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.WebLogin
+{
+    [ScreenType(PlatformName.IOS)]
+    public sealed class IosLoginScreen : LoginScreen
+    {
+        public IosLoginScreen() : base(MobileBy.AccessibilityId("Login"))
+        {
+        }
+
+        protected override By UsernameTxbLoc => MobileBy.IosNSPredicate("type == 'XCUIElementTypeTextField' AND name == 'username'");
+
+        protected override By PasswordTxbLoc => MobileBy.IosNSPredicate("type == 'XCUIElementTypeSecureTextField' AND name == 'password'");
+
```

Comment:
Why did you separate these locators with an empty line, but not the rest? Empty lines are not needed between parameters